ReactantPrototyping/ViewController.swift

```diff
@@ -21,25 +21,78 @@ final class ViewController: ControllerBase<Void, ExampleRootView> {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
-        navigationController?.hidesBarsOnTap = true
+//        navigationController?.hidesBarsOnTap = true
```

Comment:
Is this planned to be uncommented sometime in the future or is it dead code now?

Reply:
I've always considered ReactantPrototyping as almost write only playground, it'd be nice to clear it up and add testing controllers for every subspec/feature. However, there is a chance that during the time, it'll become as messy as it is now.